nsqadmin/static/js/lib/ajax_setup.js

```diff
@@ -7,6 +7,6 @@ $.ajaxPrefilter(function(options) {
         'X-UserAgent': USER_AGENT,
         'Accept': 'application/vnd.nsq; version=1.0'
     });
-    options['timeout'] = 20 * 1000;
+    options['timeout'] = 120 * 1000;
```

Comment:
I don't think we want to change this timeout. We never want to wait this long for something to load.

(I understand you need to increase it in order for it to work for your setup. I guess we'd hope to make the stats requests just not take so long somehow.)